src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/AbstractMinecraftTransformer.kt

```diff
@@ -263,8 +269,7 @@ abstract class AbstractMinecraftTransformer protected constructor(
         "*",
         "META-INF/**",
         "net/minecraft/**",
-        "com/mojang/blaze3d/**",
-        "com/mojang/realmsclient/**",
+        "com/mojang/**",
```

Comment:
This isn't correct for modern mc versions. There's a few com Mojang libraries, like authlib that don't need to remain in the merged jar